bindings/py/cpp_src/bindings/algorithms/py_SpatialPooler.cpp

```diff
@@ -410,6 +405,8 @@ Argument output An SDR representing the winning columns after
                 buf << self;
                 return buf.str(); });
 
+	py_SpatialPooler.def_property_readonly("connections", &SpatialPooler::connections, "SP's internal connections (read-only) Warning: the Connections is subject to change.");
```

Comment:
I'm having some mistake in exposing (readonly) Connections from SP to python. The same works in TM (py_TemporalMemory). I can't see what the difference could be.

Reply:
That looks correct...  I know this isn't very useful advice, but 
- maybe try rewriting this to use a callable function which gets & returns the connections? 
- Maybe try using the mutable accessor to `connections_` instead of the read-only accessor `connections`?

Reply:
> maybe try rewriting this to use a callable function which gets & returns the connections

this helped, thanks! :+1: